### DISCUSSION TITLE
[Task]: When the Android terminal controls the Android terminal, the keyboard typing delay is very large and does not follow the hand.

### DISCUSSION THREAD

**Discussion by nbgfmymtnhbg** [rustdesk/rustdesk#6494] [General]
### Are you our team member?

- [X] Yes, I am?

### SubTasks

When the Android terminal controls the Android terminal, the keyboard typing delay is very large and does not follow the hand.
![yanshi](https://github.com/rustdesk/rustdesk/assets/108757740/bbba98c7-9d9a-4b1d-9e4b-79c5877f7e69)

**Comment by 21pages** [comment]
When controlling one Android device with another Android device, there are two issues:

1.  When using the local software keyboard, most keys cannot work, except for the Enter key.

2.  When using the software keyboard of the controlled device, there are issues with  mouse events sending order. 
slow tap:  tap "1" "2" get "1" "2“
![slow_tap](https://github.com/rustdesk/rustdesk/assets/14891774/c286b2fc-fb26-4fe6-90e1-a18710babcaf)
quick tap: tap "1" "2" get "2" "2“
![quick_tap](https://github.com/rustdesk/rustdesk/assets/14891774/64035f36-4c35-4b60-afe7-7f88437a8115)
